src/main/resources/ListedElectronicMonograph.json

```diff
@@ -3,10 +3,7 @@
       "mappings": [
         {
           "field": "ACCESS_PROVIDER",
-          "dataSource": {
-            "from": "//PurchaseOption/VendorPOCode",
-            "translation": "lookupOrganization"
-          }
```

Comment:
I do not see this field at all in ticket, for this mapping ListedElectronicMonograph, so we should felete it completely

Reply:
Also it seems we need corresponding ui ticket to remove it from ui-gobi-settings
https://github.com/search?q=repo%3Afolio-org%2Fui-gobi-settings%20ACCESS_PROVIDER&type=code